tests/test_dle.py

```diff
@@ -313,6 +313,51 @@ def test_cache_blocking_imperfect_nest(blockinner):
     assert np.all(v.data == v1.data)
 
 
+@pytest.mark.parametrize("blockinner", [False, True])
+def test_cache_blocking_imperfect_nest_v2(blockinner):
+    """
+    Test that a non-perfect Iteration nest is blocked correctly. This
```

Comment:
Maybe use either imperfect or on-perfect for consistency? I guess imperfect is the better?

Reply:
it's a test docstring, so... if I'll have to touch again this branch, I will